internal/scti/handlers.go

```diff
@@ -305,7 +305,7 @@ func addChainInternal(ctx context.Context, opts *HandlerOptions, log *log, w htt
 		if err != nil {
 			if errors.Is(err, tessera.ErrPushback) {
 				w.Header().Add("Retry-After", "1")
-				return http.StatusServiceUnavailable, fmt.Errorf("Tessera sequencer pushed back: %v", err)
+				return http.StatusServiceUnavailable, fmt.Errorf("pushed back from Tessera sequencer: %v", err)
```

Comment:
nit, received pushback from Tessera sequencer. Also, we should potentially revisit this: this reveals information about the internals of the server, but doesn't go through `sendHTTPError`.

Reply:
Updated to "received pushback from Tessera sequencer".

Right, the `MaskInternalErrors` option is being ignored here.

Reply:
Created https://github.com/transparency-dev/static-ct/issues/229 to follow up later.